ssm/ssm2.py

```diff
@@ -204,7 +219,15 @@ def on_receipt(self, headers, unused_body):
         '''
         log.info('Broker received message: %s', headers['receipt-id'])
         self._last_msg = headers['receipt-id']
-        
+
+    def on_receiver_loop_completed(self, _unused_headers, _unused_body):
+        '''
```

Comment:
Triple double quotes for new code. See [PEP 8](https://www.python.org/dev/peps/pep-0008/).